src/sources/host_metrics/network.rs

```diff
@@ -15,11 +15,22 @@ use super::{filter_result, FilterList, HostMetrics};
 #[configurable_component]
 #[derive(Clone, Debug, Default)]
 pub struct NetworkConfig {
-    /// Lists of device name patterns to include or exclude.
+    /// Lists of device name patterns to include or exclude in gathering
+    /// network utilization metrics.
+    ///
+    /// Defaults to including all devices.
```

Comment:
Is this line needed? The docs should show a `default` based on the code's default. 
<img width="103" alt="image" src="https://user-images.githubusercontent.com/11903873/212937357-1c54cea8-e509-4d01-aede-97589a574ded.png">

That's from the existing site, the preview didn't seem to build though.

Reply:
We need it because the code considers a lack of config options to default to all, so we actually don't want serde to pass anything in. We could always refactor to switch that behavior.

Reply:
Actually, you're right here. I'll switch back to the previous comment.